defaults/main.yml

```diff
@@ -6,38 +6,57 @@
 ###############################################
 
 cis_apply_level_1_profile: true    # Whether Level 1 of the benchmark should be applied
-cis_apply_level_2_profile: false   # Whether Level 2 of the benchmark should be applied
+cis_apply_level_2_profile: true   # Whether Level 2 of the benchmark should be applied
 
-cis_level_1_exclusions: []  # A list of Level 1 recommendations to exclude (i.e. ['1.1.1.1'])
-cis_level_2_exclusions: []  # A list of Level 2 recommendations to exclude
+# A list of Level 1 recommendations to exclude (i.e. ['1.1.1.1'])
+# DO NOT modify it here, but in vars/
+cis_level_1_exclusions: []
+
+# A list of Level 2 recommendations to exclude
+# DO NOT modify it here, but in vars/
+cis_level_2_exclusions: []
 
 # Whether to fail when remediation items are found for recommendations which can't be automatically fixed.
 # If false, a debug message will be generated instead, with the preface *** ACTION REQUIRED ***..
 fail_on_manual_remediation_actions: false  # True or false.
 
+cis_target_os_distribution: "Amazon"
+cis_target_os_versions:
+  - "2016.03"
+  - "2016.09"
+  - "2017.03"
+  - "2017.09"
+  - "2017.12"
```

Comment:
Should `2018.03` be here?

Reply:
If anything we should remove all entries but Karoo - we no longer target Amazon Linux 1.
Karoo would cover all point releases of Amazon Linux 2.

Reply:
I agree that AL1 releases should go. This is specifically targeting AL2.